lib/getSpawnedTask.js

```diff
@@ -165,15 +167,19 @@ export const getSpawnedTask = ({
 
     try {
       const subprocess = spawn(cmd, isFn ? args : args.concat(files), spawnOptions)
-      quitInterruptCheck = interruptExecutionOnError(ctx, subprocess)
+      if (!continueOnError) {
+        quitInterruptCheck = interruptExecutionOnError(ctx, subprocess)
+      }
       const result = await subprocess
       if (verbose) {
         handleOutput(command, result, ctx)
       }
     } catch (error) {
       throw makeErr(command, error, ctx)
     } finally {
-      await quitInterruptCheck()
+      if (quitInterruptCheck) {
+        await quitInterruptCheck()
+      }
```

Comment:
We could probably use fancier syntax here:

```suggestion
      await quitInterruptCheck?.()
```